package.json

```diff
@@ -90,5 +90,11 @@
     "printWidth": 120,
     "semi": true,
     "singleQuote": true
+  },
+  "jest": {
+    "rootDir": "src",
+    "testMatch": [
+      "**/?(*.)test.js"
+    ]
```

Comment:
narrows the search space for test files, which should make jest a little faster to start up.